tests/incorrect.bib

```diff
@@ -0,0 +1,17 @@
+@article{rashid2016,
+  title={Applications of wireless sensor networks for urban areas: A survey},
+  author={Rashid, Bushra and Rehmani, Mubashir Husain},
+  journal={Journal of network and computer applications},
+  volume={60},
+  pages={192--219},
+  year={2016},
+  publisher={Elsevier}
+}
+
+@inproceedings{conigliocorbalan,
+  author    = {Marcelo Coniglio and Mar{\'{\i}}a I. Corbal{\'{a}}n},
+  title     = {Sequent Calculi for the classical fragment of {B}ochvar and {H}alld{\'{e}}n's {N}onsense {L}ogics},
+  booktitle = {Proceedings Seventh Workshop on Logical and Semantic Frameworks, with Applications, {LSFA} 2012, Rio de Janeiro, Brazil, September 29-30, 2012.},
+  pages     = {125--136},
+  year      = {2012},
```

Comment:
Maybe add a comment explaining why this file is malformed

(it's the stray comma, isn't it?)